RSDKv3/Debug.hpp

```diff
@@ -20,7 +20,14 @@ inline void PrintLog(const char *msg, ...)
         char pathBuffer[0x100];
 #if RETRO_PLATFORM == RETRO_OSX || RETRO_PLATFORM == RETRO_UWP
         if (!usingCWD)
+#if RETRO_PLATFORM == RETRO_OSX
+        {
+            getResourcesPath(pathBuffer, sizeof(pathBuffer));
+            sprintf(pathBuffer, "%s/log.txt", pathBuffer);
```

Comment:
Using the same buffer as both input and output for `sprintf` is unsafe and can cause issues, please use a temporary buffer for this.

Reply:
How would I make a temporary buffer?